CDP4CrossViewEditor/ViewModels/WorkbookSelectionDialogResult.cs

```diff
@@ -46,14 +46,17 @@ public class WorkbookSelectionDialogResult : BaseDialogResult
         /// <param name="workbook">The <see cref="Workbook"/> that is the result of a selection.</param>
         /// <param name="workbookElements">Elements that has been selected <see cref="ElementDefinition"/></param>
         /// <param name="workbookParameterTypes">Parameter types that has been selected <see cref="ParameterType"/></param>
+        /// <param name="workbookChangedValues"></param>
         public WorkbookSelectionDialogResult(bool? res, Workbook workbook,
             IEnumerable<ElementDefinition> workbookElements,
-            IEnumerable<ParameterType> workbookParameterTypes)
+            IEnumerable<ParameterType> workbookParameterTypes,
+            Dictionary<string, string> workbookChangedValues)
```

Comment:
This parameter name should be consistent with the other uses throughout the code (`WorkbookOperator:67` and ``CrossviewWorkbookdata:141``).

Maybe rename to `ManuallySavedParameterValues`.